README.md

```diff
@@ -79,6 +79,17 @@ BPKShadow *shadow = [BPKShadow shadowSm];
 [shadow applyToLayer:myView.layer];
 ```
 
+### Backpack/Gradient
+
+`Backpack/Gradient` contains the Backpack gradient definitions in the class `BPKGradient`. Add `Backpack/Gradient` to your `Podfile` or `podspec` and then use it as follows.
+
+```objective-c
+#import <Backpack/Gradient.h>
+
+BPKGradient *gradientDefault = [BPKGradient gradient];
```

Comment:
We need to mention here that it has the two properties `colors`of type `NSArray<UIColor *> *` and the stops encoded as CGPoints with components in the range 0-1 encoded as `NSValue *` in `NSArray<NSValue *> *`